source/getting-started/index.md

```diff
@@ -1,52 +1,55 @@
 Getting started with Ember.js is easy. Ember.js projects are created and managed
-through our [command line build tool ember-cli](http://www.ember-cli.com/). The command line tool brings:
-
-* Modern application asset management (including combining, minifying, and versioning).
-* Built-in generators to help you create components, routes, and more.
-* A conventional project layout so approaching other developers' Ember.js applications is easy.
-* Official [JavaScript modules](http://jsmodules.io/) to keep your project organized.
-* A complete testing framework.
-* Access to a growing ecosystem of [Ember Addons](http://www.emberaddons.com/).
-
+through our command line build tool [Ember CLI](http://www.ember-cli.com/).
+This tool provides:
+
+* Modern application asset management (including concatenation, minifying, and versioning)
+* Generators to help you create components, routes, and more
+* A conventional project layout, so approaching existing Ember.js applications is easy
+* Support for ES2015/ES6 JavaScript via the [Babel](http://babeljs.io/docs/learn-es2015/)
+  project. This includes support for JavaScript modules, which are used
```

Comment:
jsmodules looks very confusing to me. Even as someone who knows the syntax already, I was very lost when I loaded the page up:

![](https://api.monosnap.com/rpc/file/download?id=uJqE3n5q8GukTr1Ru2TvszXDVFpqqK)

How about http://exploringjs.com/es6/ch_modules.html from Alex? It is a much more focused introduction IMO, avoiding much of the comparison to CJS that plagues jsmodules. It also goes into plenty of detail further on, allowing more curious readers to keep on the deep dive.